graylog2-web-interface/src/components/event-definitions/hooks/useEventDefinitionConfigFromLocalStorage.test.ts

```diff
@@ -26,23 +25,25 @@ import {
   urlConfigWithoutAggString,
 } from 'fixtures/createEventDefinitionFromValue';
 
-jest.mock('routing/useQuery', () => jest.fn());
+jest.mock('routing/useQuery');
 
-const wrapper = ({ children }) => (
-  <div>
-    {children}
-  </div>
-);
+describe('useEventDefinitionConfigFromLocalStorage', () => {
+  beforeEach(() => {
+    asMock(useQuery).mockReturnValue({ 'session-id': 'session-id' });
+  });
+
+  afterEach(() => {
+    localStorage.removeItem('session-id');
```

Comment:
Although it is just a test, here you can use `Store` as well. It makes it easier to replace `localStore` in the future.